session_0/word_processing.rb

```diff
@@ -34,11 +48,32 @@ def words_with_prefix(prefix, words)
 # The function `similarity_score` takes two words and returns the
 # similarity score (an integer).
 def similarity_score(word_1, word_2)
-  raise NotImplementedError # TODO
+  commonPrefix=''
+  i=0
+  while i < word_1.size do
+    if word_1[i]==word_2[i]
+      commonPrefix+=word_1[i]
+    else
+      break
+    end
+    i+=1
+  end
+  commonPrefix.size
 end
 
 # Given a chosen word and an array of words, return an array of word(s)
 # with the maximum similarity score in the order they appear.
 def most_similar_words(chosen_word, words)
-  raise NotImplementedError # TODO
-end
+  maxScore=0
```

Comment:
Ruby follows `snake_case` for variable names. Please follow for other variables too